package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "build": "pnpm -- turbo run build --filter=docs",
+    "build": "pnpm -- turbo run build",
```

Comment:
Vercel settings are not using this, so it's safe to change. Vercel UI already has a custom build command